src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -151,7 +151,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
    * @param {ApartmentApplicant} applicant The applicant to be checked
    * @return {Boolean} True if valid, otherwise false
    */
-  const isApplicantValid = async (applicant) => {
+  const isApplicantValid = (applicant) => {
```

Comment:
I understand that using `async` led to issues of False statements being read as true (or vice versa), and I thought using `await` fixed that. Why are we getting rid of both now?

Reply:
There is no need to have it be async because all the internal parts of the function are now done in a synchronous manner, so it helps clean things up (because having the unneeded `await ...` can also lead to unwanted unexcepted behaviors if it is not used correctly)

It just seems to be more reliable if you don't use async unless it makes sense with the whole context

Reply:
Because of what this function does and what it is meant to represent, is it actually preferable to not have it be async, since its sole purpose is to block the execution of the things that come after it. So it is by design a synchronous concept.

Reply:
Oh, good point!